### PR TITLE
operators grafana-operator (4.0.2)

### PR DESCRIPTION
**Pull request by NissesSenap**
Signed-off-by: Edvin Norling <edvin.norling@xenit.se>

Thanks submitting your Operator. Please check below list before you create your Pull Request.

### Updates to existing Operators

* [X] Did you create a `ci.yaml` file according to the [update instructions](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md)?
* [X] Is your new CSV pointing to the previous version with the `replaces` property if you chose `replaces-mode` via the `updateGraph` property in `ci.yaml`?
* [X] Is your new CSV referenced in the [appropriate channel](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#channels) defined in the `package.yaml` or `annotations.yaml` ?
* [X] Have you tested an update to your Operator when deployed via OLM?
* [X] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?

### Your submission should not

* [X] Modify more than one operator
* [X] Modify an Operator you don't own
* [X] Rename an operator - please remove and add with a different name instead
* [X] Modify any files outside the above mentioned folders
* [X] Contain more than one commit. **Please squash your commits.**

### Operator Description must contain (in order)

1. [X] Description about the managed Application and where to find more information
2. [X] Features and capabilities of your Operator and how to use it
3. [X] Any manual steps about potential pre-requisites for using your Operator

### Operator Metadata should contain

* [X] Human readable name and 1-liner description about your Operator
* [X] Valid [category name](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#categories)<sup>1</sup>
* [X] One of the pre-defined [capability levels](https://github.com/operator-framework/operator-courier/blob/4d1a25d2c8d52f7de6297ec18d8afd6521236aa2/operatorcourier/validate.py#L556)<sup>2</sup>
* [X] Links to the maintainer, source code and documentation
* [X] Example templates for all Custom Resource Definitions intended to be used
* [X] A quadratic logo

Remember that you can preview your CSV [here](https://operatorhub.io/preview).

--

<sup>1</sup> If you feel your Operator does not fit any of the pre-defined categories, file an issue against this repo and explain your need

<sup>2</sup> For more information see [here](https://sdk.operatorframework.io/docs/overview/#operator-capability-level)

### CODE REVIEW

**Review by HubertStefanski — APPROVED** [review]
Approving :)

File: operators/grafana-operator/4.0.2/grafana-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+    url: https://grafana-operator.domain
+  maintainers:
+  - email: pbraun@redhat.com
+    name: Peter Braun
+  - email: hstefans@redhat.com
+    name: Hubert Stefanski
+  maturity: alpha
+  provider:
+    name: Red Hat
+  replaces: grafana-operator.v4.0.0
```
Comment (pb82) [inline operators/grafana-operator/4.0.2/grafana-operator.clusterserviceversion.yaml:347]: @NissesSenap should that not be `v4.0.1`?

Comment (NissesSenap) [reply operators/grafana-operator/4.0.2/grafana-operator.clusterserviceversion.yaml]: That looks like it did the trick, nice find @pb82

**Review by HubertStefanski — APPROVED** [review]
reapproving

**Pull request merged**